src/lib/dom.generated.d.ts

```diff
@@ -1275,8 +1275,8 @@ interface PushSubscriptionOptionsInit {
 }
 
 interface QueuingStrategy<T = any> {
-    highWaterMark?: number;
-    size?: QueuingStrategySize<T>;
+    readonly highWaterMark?: number;
+    readonly size?: QueuingStrategySize<T>;
```

Comment:
I'm not sure we can take this until the generator is updated to do this, right?

Reply:
Right, this ultimately needs to be done in the DOM generator. However, without this workaround it's pretty much impossible to compile anything because `lib.dom.d.ts` fails to check properly with `--enforceReadonly`:

```ts
../../ts/built/local/lib.dom.d.ts(3236,11): error TS2430: Interface 'ByteLengthQueuingStrategy' incorrectly extends interface 'QueuingStrategy<ArrayBufferView>'.
  Property 'highWaterMark' is 'readonly' in the source but not in the target.
../../ts/built/local/lib.dom.d.ts(5900,11): error TS2430: Interface 'CountQueuingStrategy' incorrectly extends interface 'QueuingStrategy<any>'.
  Property 'highWaterMark' is 'readonly' in the source but not in the target.
```